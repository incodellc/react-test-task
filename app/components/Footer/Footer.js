import React from 'react';
import './Footer.scss';

const Footer = () => {
    return (
        <div className="stock-ticker__footer">
            <div className="stock-ticker__footer-content"> Â© 2020 London Stock Exchange. All rights reserved.</div>
        </div>
    );
};

export default Footer;
