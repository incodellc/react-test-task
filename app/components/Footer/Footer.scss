@import "../variables";

.stock-ticker {
  &__footer {
    position: absolute;
    bottom: 0;
    width: 100%;
    height: 100px;
    display: flex;
    align-items: center;

    @media (min-width: $layout-breakpoint-small) {
      height: 200px;
    }

    &-content {
      padding-left: 10%;
      padding-right: 10%;
      font-size: 1.1em;
      color: $text;
      @media (min-width: $layout-breakpoint-small) {
        font-size: 1.4em;
      }
    }
  }
}