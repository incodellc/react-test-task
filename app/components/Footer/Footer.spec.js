import React from 'react';
import {shallow} from 'enzyme';
import Footer from './Footer';

it('should contain .stock-ticker__footer', () => {
    const wrapper = shallow(<Footer/>);
    expect(wrapper.find('.stock-ticker__footer').length).toBe(1);
});

it('should render Footer component', () => {
    const component = shallow(<Footer/>);
    expect(component).toMatchSnapshot();
});
