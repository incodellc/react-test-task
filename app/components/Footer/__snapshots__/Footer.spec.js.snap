// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer component 1`] = `
<div
  className="stock-ticker__footer"
>
  <div
    className="stock-ticker__footer-content"
  >
     Â© 2020 London Stock Exchange. All rights reserved.
  </div>
</div>
`;
